en/vespa8-geo-migration-guide.md

```diff
@@ -0,0 +1,84 @@
+---
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Position fields - Vespa 8 migration"
+---
+
+Refer to [Vespa 8 release notes](vespa8-release-notes.html) -
+this is a guide on how to migrate from Vespa 7 to Vespa 8 when using position fields.
+The guide is relevant for applications having a `position` field in a schema.
+
+
```

Comment:
consider adding: "for the rest of this document, we assume a schema containing... field myfield type position {...}"